docs/source/packs.rst

```diff
@@ -133,6 +133,10 @@ version or **upgrade to latest** if the version is not specified. Your config fi
 overwritten, so you can revert to an older version just as easily, but for production deployments
 we recommend to always specify versions in case there are major changes in ``latest``.
 
+.. note::
+
+    New feature! As of StackStorm 3.4. It is also possible to download and install packs from GitLab using a SSH based URL through the web interface in the Actions section.
```

Comment:
Looks like the section promises a new feature installing pack from gitlab server via Web UI only, while it's just a new format for pack install.

The documentation refers to https://github.com/StackStorm/st2/pull/5050 change and I think it just worth providing an example with `gitlab@` syntax somewhere in the "Installing a Pack" section above.

Reply:
@armab @asthLucas I've updated where the gitlab url is mentioned in the doc based on review comment. 
I've based the example on what is in https://github.com/StackStorm/st2/pull/5050 PR. 
However I have to say that I have been using the git@ successfully with gitlab, so I'm not sure what scenarios require gitlab@.